packages/e2e-tests/scripts/validate-incident-verifiers.ts

```
@@ -0,0 +1,133 @@
+            worktree,
+        );
+    } finally {
+        const removed = git(
+            ["worktree", "remove", "--force", worktree],
+            REPO_ROOT,
+        );
+        rmSync(parent, { recursive: true, force: true });
+        if (removed.status !== 0) {
+            throw new Error(
+                `could not remove trusted verifier worktree: ${removed.stderr.trim() || `git worktree remove exited ${removed.status}`}`,
+            );
```

Biome (2.5.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)